src/components/button.scss

```
@@ -0,0 +1,63 @@
+$primaryColorStop1: #16222A;
+$primaryColorStop2: #3A6073;
+$secondaryColorStop1: #9b989471;
+$secondaryColorStop2: #fdfbfaef;
+$hoverPrimaryColorStop1:#0d151a;
+$hoverPrimaryColorStop2: #2b4755;
+$hoverSecondaryColorStop1: #b8b4b2e3;
+$hoverSecondaryColorStop2: #fdfbfa36;
+$boxShadowColor: #fcf8f8;
+$disableBackgroundColor:#fff;
+$disableColor:#bababa;
+
+@mixin btn-style{
+  margin: 10px;
+  padding: 15px 35px;
+  text-align: center;
+  text-transform: uppercase;
+  transition: 0.5s;
+  background-size: 200% auto;            
+  box-shadow: 0 0 20px $boxShadowColor;
+  border-radius: 10px;
+  display: inline-block;
+}
+
+.button-Primary {
+  @include btn-style;
+    background-image: linear-gradient($primaryColorStop1 70%, $primaryColorStop2  100%);
+    border: 1px solid $primaryColorStop1;
+    color: $secondaryColorStop2;
+  }
+
+.button-Secondary {
+  @include btn-style;
+    background-image: linear-gradient($secondaryColorStop1 70%, $secondaryColorStop2  100%);
+    border: 1px solid $secondaryColorStop1;
+    color: $primaryColorStop1;
+  }
+    
+.button-Primary:hover,
+.button-Primary:active { 
+  background-image: linear-gradient($hoverPrimaryColorStop1 70%, $hoverPrimaryColorStop2 100%);
+  color: $secondaryColorStop2;
+  text-decoration: none;
+}
+  
+.button-Secondary:hover,
+.button-Secondary:active {
+  background-image: linear-gradient($hoverSecondaryColorStop1 70%, $hoverSecondaryColorStop2 100%);
+  color: $primaryColorStop1;
+  text-decoration: none;
+}
+
+.button-Primary:disabled,
+.button-Secondary:disabled {
+  background-color: $disableBackgroundColor;
+  border-color: $disableColor;
+  color: $disableColor;
+  box-shadow: none;
+  opacity: 0.7;
+  cursor: not-allowed;
+  border: 1px solid;
```

```suggestion
  border: 1px solid $disableColor;
```

and remove `border-color: $disableColor;`